api/specs/webserver/openapi-projects.yaml

```diff
@@ -395,6 +395,43 @@ paths:
         default:
           $ref: "./openapi.yaml#/components/responses/DefaultErrorResponse"
 
+  /projects/{project_id}/nodes/{node_id}/retrieve:
```

Comment:
API standard with ``:`` as ``/projects/{project_id}/nodes/{node_id}:retrieve``

Reply:
It is already like that. Please have a look [here](https://github.com/ITISFoundation/osparc-simcore/pull/2509/files#diff-88c482a422bb66a12aa9358bf65e271bc6ee97a864c8b333251c2eaccc0d1775R203). In the context of this file column `:` will not work.